tutorial/source/forecasting_i.ipynb

```diff
@@ -0,0 +1,926 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Forecasting: univariate, heavy tailed\n",
+    "\n",
+    "This tutorial introduces the [pyro.contrib.forecast](http://docs.pyro.ai/en/latest/contrib.forecast.html) module, a framework for forecasting with Pyro models. This tutorial covers only univariate models and simple likelihoods. This tutorial assumes the reader is already familiar with [SVI](http://pyro.ai/examples/svi_part_ii.html) and [tensor shapes](http://pyro.ai/examples/tensor_shapes.html).\n",
+    "\n",
+    "#### Summary\n",
+    "- To create a forecasting model:\n",
```

Comment:
IIRC, you might need to add an empty line; so that sphinx can render lists correctly.